packages/common/src/store/cache/reducer.ts

```diff
@@ -100,9 +100,6 @@ export const mergeCustomizer = (objValue: any, srcValue: any, key: string) => {
     return srcValue
   }
 
-  if (key === 'artwork') {
-    return null
-  }
```

Comment:
@dylanjeffers Seb and I had a question about this. Wondering if there was a specific reason that we needed to remove artwork from the track that we received from the backend?

Reply:
This is needed for the image to be displayed immediately

Reply:
i dont think this comes from backend, the idea is to remove it if it exists in frontend from editing i think. does it negatively affect what this pr is trying to do?

Reply:
when requesting the track on the server side, this field is present so i assumed it was already on there and fine to use

Reply:
Oh interesting! I don't think that's normally a field from the api right?

Reply:
I'm not sure if it was always there bc there was a comment about unused fields, but it is a field on the tracks model and was consistently coming back when requesting the track from the sdk

Reply:
just checked it looks like it is in fact a field from sdk! so how does that get generated from the backend? if the backend now handles the rendevous algorithm for us that is really nice, mind investigating? that does make things a bit harder when coercing this sdk response to something that works with our current cache, since artwork is specifically used for "upload", "edit" etc.

Reply:
Ah that's good context on artwork being used for the upload/edit forms! And yeah I'm also very curious when artwork started coming back from the sdk, agreed it has a big impact on our handling of images in general